xml/yast2_ncurses.xml

```diff
@@ -158,16 +138,16 @@
     </listitem>
    </varlistentry>
    <varlistentry>
-    <term>Using Navigation Tree in ncurses Mode</term>
+    <term>Using Navigation Tree</term>
```

Comment:
Using the Navigation Tree? (not sure)

Reply:
Articles can be omitted in titles. But I added _the_ for the sake of consistency.